Money*/Money*/Scenes/Signup/SignupViewController.swift

```diff
@@ -33,4 +34,36 @@ final class SignupViewController: UIViewController {
         [createButton,
          cancelButton].forEach { $0?.makeRoundedAndShadowed() }
     }
+    
+    // MARK: - IBActions
+    @IBAction private func onCreateButtonTouchUpInside(_ sender: Any) {
+        guard let email = emailTextField.text,
```

Comment:
anh nên tạo 1 struct chứa các func để validate email và func để validate password sau đó gọi để sử dụng, như vậy sẽ tăng tính reuse của code hơn

Reply:
vì email, password, số điện thoại,... đều có các pattern validate nên hoàn toàn có thể tách ra như vậy đc